.travis.yml

```diff
@@ -10,4 +10,4 @@ rvm:
 script:
 - bundle exec rubocop
 - bundle exec foodcritic -f any .
-- bundle exec rspec test/unit
```

Comment:
this needs to stay unless you move the specs from test/unit to the default spec directory.